packages/libs/eda/src/lib/map/analysis/TimeSliderQuickFilter.tsx

```diff
@@ -406,10 +406,9 @@ export default function TimeSliderQuickFilter({
                   constraints={timeSliderVariableConstraints}
                 />
               </div>
-              <div>
+              <div style={{ marginRight: 'auto', marginLeft: 'auto' }}>
```

Comment:
Could you try it like this too please, and post some screenshots? (see other suggestion below)

```suggestion
              <div style={{ marginLeft: 'auto' }}>
```

Reply:
Oh, so they are too close now? I hoped they'd be a bit closer. Maybe a fixed 10px margin on the side closest to the date pickers?

Reply:
@bobular Aha I see. Then we may set value instead of "auto". For example
- removing marginRight or marginLeft
![2024-08-21 11 50 36](https://github.com/user-attachments/assets/09ee0dbb-ec98-4a06-b350-2d42fee8381d)

- set "1em": I think it would be either: Didn't check the parent's css carefully but usually 1em = 12 or 16px
![2024-08-21 11 49 38](https://github.com/user-attachments/assets/ddaa94a5-eb9e-4354-b4f7-1ac43ae3fca8)

Reply:
Looks great with 1em!

Reply:
(yes, one auto, one fixed)